ontology/yaml/resources/HVAC/entity_types/FAN.yaml

```diff
@@ -283,35 +283,69 @@ FAN_SS_DRSM8X:
   - DRSM8X
 
 FAN_EDPM_ZHM_ZTM_LPS:
-  id: "16288085548559433728"
   description: "Exhaust fan with sensors (temperature, humidity and low pressure)."
   is_canonical: true
   implements:
   - FAN
   - EDPM
   - ZHM
   - ZTM
-  uses:
-  - run_command
-  - run_status
+  opt_uses:
+  - exhaust_fan_run_command
+  - exhaust_fan_run_status
   - control_mode
   - low_differential_pressure_alarm
-  opt_uses:
   - lost_power_alarm
 
-FAN_LPS:
-  id: "16288085548559433728"
+
+FAN_FCS:
   description: "Exhaust fan with sensors (temperature, humidity and low pressure)."
   is_canonical: true
   implements:
   - FAN
-  uses:
-  - run_command
-  - run_status
+  opt_uses:
+  - lost_power_alarm
   - control_mode
   - low_differential_pressure_alarm
+  - schedule_run_command
+  - exhaust_fan_run_command
+  - exhaust_fan_run_status
+
+FAN_SA:
```

Comment:
Supply fan is not necessary for the field descriptions, and so this is really a `FAN_SS_VSC` with some optional fields. Please change.

Reply:
Type removed.